src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int firstLeg;
+    private final int height;
+    private final int secondLeg;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea((firstLeg * secondLeg) / 2 * height);
+    }
```

Comment:
The method 'obtainTheArea' is not named properly according to its purpose. This method calculates the area, so a better name would be 'calculateArea'. Also, the formula for the area of an isosceles trapezoid should be '(firstLeg + secondLeg) / 2 * height', not '(firstLeg * secondLeg) / 2 * height'.